integration-cli/test_vars_windows.go

```diff
@@ -0,0 +1,8 @@
+// +build windows
```

Comment:
While porting docker cli to windows we decided to explicitly keep those tags explicitly even though file names do the same thing.